libminifi/test/Test.cpp

```diff
@@ -1,6 +1,4 @@
 /**
- * @file MiNiFiMain.cpp 
```

Comment:
Ah yeah this one is per another PR I have. I'll remove it here since the other PR would need to be rebased, anyway.